web/stripe-template/api/routes/GET-subscription-callback.ts

```diff
@@ -1,33 +1,34 @@
 import { RouteContext } from "gadget-server";
 import { stripe } from "../stripe";
 
-/**
- * Route handler for GET subscription-callback
- *
- * @param { RouteContext } route context - see: https://docs.gadget.dev/guides/http-routes/route-configuration#route-context
- */
 export default async function route({
   request,
   reply,
   api,
   logger,
   connections,
   currentAppUrl,
-}) {
-  const { query } = request;
+}: RouteContext) {
+  const { session_id, user_id } = request.query as {
+    session_id: string;
+    user_id: string;
+  };
```

Comment:
I think this should look like
```
const route: RouteHandler<{
  Querystring: {
    session_id: string;
    user_id: string;
  };
}> = async ({
  request,
  reply,
  api,
  logger,
  connections,
  currentAppUrl,
}) => {
    const { session_id, user_id } = request.query;
```

and then we export the default route

Reply:
Yeah, that worked